http4s/src/main/scala/clue/http4s/Http4sWSBackend.scala

```diff
@@ -29,14 +27,14 @@ final class Http4sJDKWSBackend[F[_]: Async](client: WSClient[F]) extends WebSock
   ): F[PersistentConnection[F, WebSocketCloseParams]] =
     client
       .connectHighLevel(
-        WSRequest(uri, headers = Headers("Sec-WebSocket-Protocol" -> "graphql-ws"))
+        WSRequest(uri).withHeaders(Headers("Sec-WebSocket-Protocol" -> "graphql-ws"))
       )
-      .allocated
+      .allocated // TODO replace with allocatedCase
```

Comment:
CE 3.4.0 will include an `allocatedCase` method so that you can call `release` with the correct `ExitCase`.